openlibrary/templates/publishers/view.html

```diff
@@ -12,7 +12,7 @@ <h1>
         <span class="count" id="coversCount">
             <em>Publisher</em> - <strong><span>$ungettext("%(count)d work", "%(count)d works", page.work_count, count=page.work_count)</span></strong>
             $if page.ebook_count > 0:
-                / <span class="ebookcount"><span id="ebooks">$ungettext("%(count)s ebook", "%(count)s ebooks", page.ebook_count, count=commify(page.ebook_count))</span></span>
+                / <span class="ebookcount"><span id="ebooks">$ungettext("%(count)d ebook", "%(count)d ebooks", page.ebook_count, count=commify(page.ebook_count))</span></span>
```

Comment:
Hmm I think we want this as a string; `commify` converts a number to a string (eg `3871` -> `3,871`).

```suggestion
                / <span class="ebookcount"><span id="ebooks">$ungettext("%(count)s ebook", "%(count)s ebooks", page.ebook_count, count=commify(page.ebook_count))</span></span>
```

Reply:
OOPS SORRY